### DISCUSSION TITLE
Combining Forms

### DISCUSSION THREAD

**Discussion by Schlaegel** [character-map-uwp/Character-Map-UWP#288] [Ideas]
Is there a way to see and export a combining combination as SVG?  I can enter the combination in the search box and it shows there, but finds nothing. I can't see any way to enter the combination.

**Comment by Schlaegel** [comment]
See
https://en.wikipedia.org/wiki/Combining_character#:~:text=In%20digital%20typography%2C%20combining%20characters,marks%20(including%20combining%20accents).

And
https://emojipedia.org/emoji-zwj-sequence

**Comment by JohnnyWestlake** [comment]
Related to https://github.com/character-map-uwp/Character-Map-UWP/discussions/269, probably will not happen any time soon as reading the required tables to find out exactly what combinations a font supports is complex (it's not just a straightforward linear table read) and there aren't any good free .NET libraries I've found that can do it. 

I'm open to someone else attempting it
